daemon/container_unix.go

```diff
@@ -896,8 +900,6 @@ func (container *Container) configureNetwork(networkName, service, networkDriver
 		}
 	}
 
-	container.updateSandboxNetworkSettings(sb)
```

Comment:
Also, if we set the ID during the container setup, that would be available at pre-start hook time, meaning we don't need to do a walk of sandboxes--could just use the `netController.SandboxByID(..)` method, right?